SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -70,6 +78,7 @@ Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on to
 %prep
 
 %setup -q -n cloud-hypervisor-msft-v%{version}
+%autopatch -p0
```

Comment:
Just making sure I understand: does this mean the patch applies both to the original sources as well as the vendored sources?

Reply:
It is my understanding the vendored package only includes the /vendor directory. The patch is against Cargo.lock, so needs to be applied in both cases. And needs to be applied as part of the creating the vendor archive as well, to ensure that the updated version of openssl dependency gets cached.